notebooks/metatranscriptomics_pipeline_iSeqnorm.ipynb

```diff
@@ -0,0 +1,1477 @@
+    "    Set to 0 to pool all samples, regardless of concentration. Increasing this will have the \n",
+    "    effect of increasing pool concentration, at the expense of samples dropping out. \n",
+    ">   - **floor_conc**: This is the lowest concentration equivalent for which a sample will be \n",
+    "    accurately pooled. Samples below this concentration will be pooled to the volume that they \n",
+    "    would have been if they were actually that concentration. For example, if `floor_conc=20`, \n",
+    "    and a sample at 20 nM pools at 500 nL, a sample at 40 nM will pool at 250 nL but a sample at \n",
+    "    10 nM will still pool at 500 nL (rather than 1000). Increasing this value will have the effect \n",
+    "    of increasing pool concentration, but decreasing read counts for low-concentration samples. \n",
+    ">   - **total_nmol**: This is the total number of molecules to shoot for in the pool. Increasing\n",
+    "    this will increase the overall volume of the pool.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Calculate and plot pooling volumes"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "threshold = find_threshold(plate_df['MiniPico Library Concentration'], plate_df['Blank'])\n",
+    "plate_df = autopool(plate_df,method='evp',total_vol=190)\n",
+    "\n",
+    "# visualize\n",
+    "print(\"Floor concentration: {}\".format(threshold))\n",
+    "vols = make_2D_array(plate_df, data_col='MiniPico Pooled Volume', well_col=well_col).astype(float)\n",
+    "conc, vol = estimate_pool_conc_vol(plate_df['MiniPico Pooled Volume'], plate_df['MiniPico Library Concentration'])\n",
+    "print(\"Pool concentration: {:.2f}\".format(conc))\n",
+    "print(\"Pool volume: {:.2f}\".format(vol))\n",
+    "with suppress(np.linalg.LinAlgError):\n",
+    "    plot_plate_vals(vols)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "vols = make_2D_array(plate_df, data_col='MiniPico Pooled Volume', well_col=well_col).astype(float)\n",
+    "sns.scatterplot(x='MiniPico Library Concentration', y='MiniPico Pooled Volume',data=plate_df)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 5: write pooling pick list"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Write the picklist as .csv\n",
+    "picklist_fp = './test_output/Pooling/YYYY_MM_DD_NPH_3-6_evp.csv'\n",
+    "\n",
+    "if os.path.isfile(picklist_fp):\n",
+    "    print(\"Warning! This file exists already.\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "picklist = format_pooling_echo_pick_list(vols, max_vol_per_well=30000)\n",
```

Comment:
It would be helpful to add a comment explaining where the max_vol_per_well=30000 hardcoding came from.

Reply:
will add an explanation.